File: backend/src/main/java/com/yagubogu/member/dto/MemberFavoriteResponse.java
package com.yagubogu.member.dto;

import com.yagubogu.team.domain.Team;

public record MemberFavoriteResponse(
        String favorite
) {

    public static MemberFavoriteResponse from(final Team team) {
        return new MemberFavoriteResponse(team.getShortName());
    }
}

File: android/app/src/main/java/com/yagubogu/data/dto/response/member/MemberFavoriteResponse.kt
package com.yagubogu.data.dto.response.member

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class MemberFavoriteResponse(
    @SerialName("favorite")
    val favorite: String? = null, // 팀 이름
)
